### ISSUE TITLE
Results limited to 80 per page

### ISSUE THREAD

**Issue by phillc73** [samjbmason/punkapi#55]
The API V2 documentation page says:

_All parameters are optional and without them the API will just return the beers in ascending order from their ID._ 

You also mentioned in [this issue](https://github.com/samjbmason/punkapi-site/issues/5) that:

_...you can return as many or as few as you like from the api..._

Therefore, I did think that simply querying the base URL would return all beers. However, this:

```
curl https://api.punkapi.com/v2/beers
```

Returns only twenty five results, which seems to be the default `per_page` number.

Using the `per_page` parameter seems to max out at 80

```
curl https://api.punkapi.com/v2/beers?per_page=81
{"statusCode":400,"error":"Bad Request","message":"Invalid query params","data":[{"location":"query","param":"per_page","msg":"Must be a number greater than 0 and less than 80","value":"81"}]}
```

I've just started writing a wrapper for the API and am trying to handle the case where the user inputs no parameters.

This could also be linked to Issues #44 and #52

**Comment by sammdec** [comment]
Yeah I think as the db grows I would prefer to return a default amount (currently 25). I need to rewrite the server to be honest and it’s been on my todo list for a while. I ideally want to make this one massive mono repo as things are spread over several repos currently
